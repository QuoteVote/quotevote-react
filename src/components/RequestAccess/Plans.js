import React from 'react'

import Grid from '@material-ui/core/Grid'
import Link from '@material-ui/core/Link'
import Typography from '@material-ui/core/Typography'
import Card from '@material-ui/core/Card'
import CardContent from '@material-ui/core/CardContent'
import CheckIcon from '@material-ui/icons/Check'
import Radio from '@material-ui/core/Radio'
import Button from '@material-ui/core/Button'
import { makeStyles, withStyles } from '@material-ui/core/styles'

import reqAccessBusiness from 'assets/img/RequestAccess/Illustration.png'
import reqAccessPersonal from 'assets/img/RequestAccess/PersonalPlan.png'
import PropTypes from 'prop-types'

const useStyles = makeStyles(() => ({
  plansHeader: {
    height: '41px',
    objectFit: 'contain',
    font: 'Montserrat',
    fontSize: '34px',
    fontWeight: 'bold',
    letterspacing: '0.25px',
  },
  plansCardImage: {
    width: '200.6px',
    height: '140px',
    marginLeft: '10%',
  },
  plansCardHeader: {
    font: 'Montserrat',
    fontsize: 37,
    lineHeight: 1.56,
    fontWeight: 600,
    color: '#333333',
    textAlign: 'center',
  },
  plansCardText: {
    font: 'Roboto',
    fontsize: 20,
    lineHeight: 0.75,
    fontWeight: 500,
    color: '#333333',
    textAlign: 'center',
  },
  checkIconPersonal: {
    color: '#157ffb',
    marginRight: 5,
  },
  checkIconBusiness: {
    color: '#791e89',
    marginRight: 5,
  },
  link: {
    opacity: 0.8,
    font: 'Roboto',
    fontSize: 16,
    letterSpacing: 0.25,
    color: '#424556',
  },
  requestBtn: {
    textTransform: 'none',
    backgroundColor: '#00cf6e',
    color: 'white',
    position: 'absolute',
    bottom: '1%',
    left: '27%',
    '&:hover': {
      backgroundColor: '#00cf6e',
    },
  },
}))

const PersonalPlanRadio = withStyles({
  root: {
    color: '#157ffb',
    '&$checked': {
      color: '#157ffb',
    },
  },
  checked: {},
})((props) => <Radio color="default" {...props} />)

const BusinessPlanRadio = withStyles({
  root: {
    color: '#791e89',
    '&$checked': {
      color: '#791e89',
    },
  },
  checked: {},
})((props) => <Radio color="default" {...props} />)

const RequestButton = (props) => {
  const { classes, onClickRequest } = props

  return (
    <Button
      variant="contained"
      className={classes.requestBtn}
      onClick={() => onClickRequest(true)}
    >
      Request Access
    </Button>
  )
}

RequestButton.propTypes = {
  classes: PropTypes.object,
  onClickRequest: PropTypes.func,
}

const Plans = (props) => {
  const classes = useStyles()
  const { onPlanSelect, selectedPlan, setRequest } = props

  // selectedPlan can be null
  const isBusiness = selectedPlan === 'business'
  const isPersonal = selectedPlan === 'personal'

  return (
    <Grid container justify="center" style={{ marginRight: 24 }}>
      <Grid item xs={12} style={{ marginBottom: '5%' }}>
        <Typography align="center" className={classes.plansHeader}>
          What is best for you
          {' '}
          <span
            className={classes.plansHeader}
            style={{ color: '#00cf6e' }}
          >
            ?
          </span>
        </Typography>
      </Grid>
      <Grid item xs={12}>
        <Grid container justify="center" spacing={6}>
          <Grid item xs={3} style={{ marginRight: '15%', position: 'relative' }}>
            {isPersonal && <RequestButton classes={classes} onClickRequest={setRequest} />}
            <img
              alt={reqAccessPersonal}
              height={500}
              src={`${reqAccessPersonal}`}
              className={classes.plansCardImage}
            />
            <Card>
              <CardContent>
                <Grid container spacing={2}>
                  <Grid item xs={12}>
                    <Typography align="center" className={classes.plansCardHeader}>
                      Personal Plan
                    </Typography>
                  </Grid>
                  <Grid item xs={12}>
                    <Typography align="center" className={classes.plansCardText}>
                      <CheckIcon fontSize="inherit" className={classes.checkIconPersonal} />
                      Pay what you like
                    </Typography>
                  </Grid>
                  <Grid item xs={12} style={{ marginBottom: 20 }}>
                    <Typography align="center" className={classes.plansCardText}>
                      <CheckIcon fontSize="inherit" className={classes.checkIconPersonal} />
                      Free for individuals
                    </Typography>
                  </Grid>
                  <Grid item xs={12}>
                    <Typography align="center" className={classes.plansCardText}>
                      Full access
                      {' '}
                      <span
                        className={classes.plansCardText}
                        style={{
                          opacity: 0.8,
                          borderRadius: 5,
                          border: 'solid 1px #157ffb',
                          backgroundColor: '#157ffb',
                          color: 'white',
                        }}
                      >
                        for free
                      </span>
                    </Typography>
                  </Grid>
                  <Grid item xs={12}>
                    <Typography align="center" className={classes.plansCardText} style={{ color: '#157ffb' }}>
                      <PersonalPlanRadio
                        checked={isPersonal}
                        value="personal"
                        onChange={(e) => onPlanSelect(e.target.value)}
                      />
                      This works for me
                    </Typography>
                  </Grid>
                  <Grid item xs={12}>
                    <Typography align="center">
                      <Link
                        component="button"
                        href="https://"
                        className={classes.link}
                      >
                        I would like to read more
                      </Link>
                    </Typography>
                  </Grid>
                </Grid>
              </CardContent>
            </Card>
          </Grid>
          <Grid item xs={3} style={{ position: 'relative' }}>
            {isBusiness && <RequestButton classes={classes} onClickRequest={setRequest} />}
            <img
              alt={reqAccessBusiness}
              height={500}
              src={`${reqAccessBusiness}`}
              className={classes.plansCardImage}
            />
            <Card>
              <CardContent>
                <Grid container spacing={2}>
                  <Grid item xs={12}>
                    <Typography align="center" className={classes.plansCardHeader}>
                      Business Plan
                    </Typography>
                  </Grid>
                  <Grid item xs={12}>
                    <Typography align="center" className={classes.plansCardText}>
                      <CheckIcon fontSize="inherit" className={classes.checkIconBusiness} />
                      License for Professionals
                    </Typography>
                  </Grid>
                  <Grid item xs={12} style={{ marginBottom: 20 }}>
                    <Typography align="center" className={classes.plansCardText}>
                      <CheckIcon fontSize="inherit" className={classes.checkIconBusiness} />
                      &#162;10 per
                      {' '}
                      <span style={{ color: '#791e89' }}>POP</span>
                      {' '}
                      Prediction
                    </Typography>
                  </Grid>
                  <Grid item xs={12}>
                    <Typography align="center" className={classes.plansCardText}>
                      <span
                        className={classes.plansCardText}
                        style={{
                          opacity: 0.8,
                          borderRadius: 5,
                          border: 'solid 1px #791e89',
                          backgroundColor: '#791e89',
                          color: 'white',
                        }}
                      >
                        $10
                      </span>
                      {' '}
                      / month
                    </Typography>
                  </Grid>
                  <Grid item xs={12}>
                    <Typography align="center" className={classes.plansCardText} style={{ color: '#791e89' }}>
                      <BusinessPlanRadio
                        checked={isBusiness}
                        value="business"
                        onChange={(e) => onPlanSelect(e.target.value)}
                      />
                      This works for me
                    </Typography>
                  </Grid>
                  <Grid item xs={12}>
                    <Typography align="center">
                      <Link
                        component="button"
                        href="https://"
                        className={classes.link}
                      >
                        I would like to read more
                      </Link>
                    </Typography>
                  </Grid>
                </Grid>
              </CardContent>
            </Card>
          </Grid>
        </Grid>
      </Grid>
    </Grid>
  )
}
Plans.propTypes = {
  onPlanSelect: PropTypes.any,
  selectedPlan: PropTypes.any,
  setRequest: PropTypes.func,
}


export default Plans
