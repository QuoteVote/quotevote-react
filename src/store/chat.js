/* eslint-disable no-param-reassign */
import { createSlice } from '@reduxjs/toolkit'

const chat = createSlice({
  name: 'chat',
  initialState: {
    submitting: false,
    selectedRoom: null,
  },
  reducers: {
    CHAT_SUBMITTING: (state, action) => {
      state.submitting = action.payload
    },
    SELECTED_CHAT_ROOM: (state, action) => {
      state.selectedRoom = action.payload
    },
  },
})

export const { CHAT_SUBMITTING, SELECTED_CHAT_ROOM } = chat.actions

export default chat.reducer
