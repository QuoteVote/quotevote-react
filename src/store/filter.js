/* eslint-disable no-param-reassign */
import { createSlice } from '@reduxjs/toolkit'

const uiSlice = createSlice({
  name: 'filter',
  initialState: {
    filter: {
      visibility: false,
      value: ['POSTED'],
    },
    date: {
      visibility: false,
      value: '',
    },
    search: {
      visibility: false,
      value: '',
    },
  },
  reducers: {
    FILTER_VISIBILITY: (state, action) => {
      state.filter.visibility = action.payload
    },
    FILTER_VALUE: (state, action) => {
      state.filter.value = action.payload
    },
    DATE_VISIBILITY: (state, action) => {
      state.date.visibility = action.payload
    },
    DATE_VALUE: (state, action) => {
      state.date.value = action.payload
    },
    SEARCH_VISIBILITY: (state, action) => {
      state.search.visibility = action.payload
    },
    SEARCH_VALUE: (state, action) => {
      state.search.value = action.payload
    },
  },
})

export const {
  FILTER_VISIBILITY,
  FILTER_VALUE,
  DATE_VISIBILITY,
  DATE_VALUE,
  SEARCH_VISIBILITY,
  SEARCH_VALUE,
} = uiSlice.actions

export default uiSlice.reducer
