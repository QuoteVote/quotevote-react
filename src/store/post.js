/* eslint-disable no-param-reassign */
import { createSlice } from '@reduxjs/toolkit'

const postSlice = createSlice({
  name: 'post',
  initialState: {
    selectedPost: {
      id: null,
    },
  },
  reducers: {
    SET_SELECTED_POST: (state, action) => {
      state.selectedPost.id = action.payload
    },
  },
})

export const { SET_SELECTED_POST } = postSlice.actions

export default postSlice.reducer
