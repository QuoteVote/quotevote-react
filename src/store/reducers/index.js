export { default as postReducer } from './postReducer'
export { default as loginReducer } from './loginReducer'
export { default as chatReducer } from './chatReducer'
export { default as appReducer } from './appReducer'
