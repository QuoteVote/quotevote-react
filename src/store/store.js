import { combineReducers } from 'redux'
import { configureStore, getDefaultMiddleware } from '@reduxjs/toolkit'
import postReducer from 'store/post'
import userReducer from 'store/user'

const rootReducer = combineReducers({
  user: userReducer,
  post: postReducer,
})

// Add middleware the end of the array
const middleware = [...getDefaultMiddleware()]

const devMode = process.env.NODE_ENV === 'development'

export default configureStore({
  reducer: rootReducer,
  middleware,
  devTools: devMode,
})
