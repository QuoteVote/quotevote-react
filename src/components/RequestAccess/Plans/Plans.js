import React, { useState } from 'react'
import { makeStyles } from '@material-ui/core/styles'
import { tokenValidator } from 'store/user'
import { useDispatch, useSelector } from 'react-redux'
import { useHistory } from 'react-router-dom'
import styles from 'assets/jss/material-dashboard-pro-react/views/landingPageStyle'
import BusinessPlanAvatar from 'assets/img/BusinessPlanAvatar.png'
import InvestorPlanAvatar from 'assets/img/InvestorPlanAvatar.png'
import PersonalPlanAvatar from 'assets/img/PersonalPlanAvatar.png'
import { Typography, Grid, Button } from '@material-ui/core'
import { SET_SELECTED_PLAN } from 'store/ui'
import { isMobile } from 'react-device-detect'

const useStyles = makeStyles(styles)

export const MOBILE_IMAGE_WIDTH = 250

export default function Plans() {
  const classes = useStyles({ isMobile })
  const dispatch = useDispatch()
  const history = useHistory()
  const selectedPlan = useSelector((state) => state.ui.selectedPlan)

  React.useEffect(() => {
    if (tokenValidator(dispatch)) history.push('/hhsb/Home')
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [])

  const setSelectedPlan = (type) => {
    dispatch(SET_SELECTED_PLAN(type))
  }

  return (
    <div className={classes.container}>
      <Grid container justify="center" style={{ marginRight: 24 }}>
        <Grid item container justify="center" xs={12}>
          <Typography className={classes.select}>Select To Learn More</Typography>
          <Grid container direction="row" justify="space-evenly">
            <Grid item>
            <Grid container direction="column" alignItems="center">
              <img className={classes.planAvatar} src={PersonalPlanAvatar} alt="personal" />
              <Button
                className={classes.planButton}
                variant="outlined"
                onClick={() => setSelectedPlan('personal')}
              >
                Personal
              </Button>
              </Grid>
            </Grid>
            <Grid item>
            <Grid container direction="column" alignItems="center">
              <img className={classes.planAvatar} src={BusinessPlanAvatar} alt="business" />
              <Button
                className={classes.planButton}
                variant="outlined"
                onClick={() => setSelectedPlan('business')}
              >
                Business
              </Button>
              </Grid>
            </Grid>
            <Grid item>
            <Grid container direction="column" alignItems="center">
              <img className={classes.planAvatar} src={InvestorPlanAvatar} alt="investor" />
              <Button
                className={classes.planButton}
                variant="outlined"
                onClick={() => setSelectedPlan('investors')}
              >
                Investors
              </Button>
              </Grid>
            </Grid>
          </Grid>
        </Grid>
      </Grid>
    </div>
  )
}
