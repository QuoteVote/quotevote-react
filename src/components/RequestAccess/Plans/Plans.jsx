import React from 'react'
import { makeStyles } from '@material-ui/core/styles'
import { tokenValidator } from 'store/user'
import { useDispatch } from 'react-redux'
import { useHistory } from 'react-router-dom'
import styles from 'assets/jss/material-dashboard-pro-react/views/landingPageStyle'
import {
  Typography,
  Grid,
  Button,
  Hidden,
} from '@material-ui/core'
import { SET_SELECTED_PLAN } from 'store/ui'
import { isMobile } from 'react-device-detect'

const useStyles = makeStyles(styles)

export const MOBILE_IMAGE_WIDTH = 250

export default function Plans() {
  const classes = useStyles({ isMobile })
  const dispatch = useDispatch()
  const history = useHistory()

  React.useEffect(() => {
    if (tokenValidator(dispatch)) history.push('/Home')
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [])

  const setSelectedPlan = (type) => {
    dispatch(SET_SELECTED_PLAN(type))
    history.push('/auth/plans')
  }

  const buttonList = (
    <Grid container direction="column" justify="space-evenly" alignItems="center">
      <Button
        className={classes.planButton}
        variant="outlined"
        onClick={() => setSelectedPlan('personal')}
      >
        Personal
      </Button>
      <Button
        className={classes.planButton}
        variant="outlined"
        onClick={() => setSelectedPlan('business')}
      >
        Business
      </Button>
      <Button
        className={classes.planButton}
        variant="outlined"
        onClick={() => setSelectedPlan('investors')}
      >
        Investors
      </Button>
    </Grid>
  )

  return (
    <div className={classes.container}>
      <Grid container justify="center" style={{ marginRight: 24 }}>
        <Grid item container justify="center" xs={12}>
          <Typography className={classes.select}>Select To Learn More</Typography>
          <Hidden smUp>
            {buttonList}
          </Hidden>
          <Hidden xsDown>
            <Grid container direction="row" justify="space-evenly" className={classes.plans}>
              <Grid item>
                <Grid container direction="column" alignItems="center">
                  <img className={classes.planAvatar} src="/assets/PersonalPlanAvatar.png" alt="personal" />
                  <Button
                    className={classes.planButton}
                    variant="outlined"
                    onClick={() => setSelectedPlan('personal')}
                  >
                    Personal
                  </Button>
                </Grid>
              </Grid>
              <Grid item>
                <Grid container direction="column" alignItems="center">
                  <img className={classes.planAvatar} src="/assets/BusinessPlanAvatar.png" alt="business" />
                  <Button
                    className={classes.planButton}
                    variant="outlined"
                    onClick={() => setSelectedPlan('business')}
                  >
                    Business
                  </Button>
                </Grid>
              </Grid>
              <Grid item>
                <Grid container direction="column" alignItems="center">
                  <img className={classes.planAvatar} src="/assets/InvestorPlanAvatar.png" alt="investor" />
                  <Button
                    className={classes.planButton}
                    variant="outlined"
                    onClick={() => setSelectedPlan('investors')}
                  >
                    Investors
                  </Button>
                </Grid>
              </Grid>
            </Grid>
          </Hidden>
        </Grid>
      </Grid>
    </div>
  )
}
