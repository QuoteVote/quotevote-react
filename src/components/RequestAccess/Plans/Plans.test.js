import React from 'react'
import { render } from '@testing-library/react'
import { Provider } from 'react-redux'
import Plans from './Plans'
import store from '../../../store/store'

const PlansWrapper = () => (
  <Provider store={store}>
    <Plans />
  </Provider>
)

describe('Plans test -', () => {
  it('renders correctly', () => {
    const { container } = render(<PlansWrapper />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
