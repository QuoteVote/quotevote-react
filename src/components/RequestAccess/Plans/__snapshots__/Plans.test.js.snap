// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plans test - renders correctly 1`] = `
<div
  class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
  style="margin-right: 24px;"
>
  <div
    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    style="margin-bottom: 5%;"
  >
    <p
      class="MuiTypography-root makeStyles-plansHeader-8 MuiTypography-body1 MuiTypography-alignCenter"
    >
      Select to learn more
    </p>
  </div>
  <div
    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
    >
      <div
        class="MuiGrid-root makeStyles-gridContent-18 MuiGrid-item MuiGrid-grid-xs-7 MuiGrid-grid-md-3"
      >
        <img
          alt="PersonalPlan.png"
          class="makeStyles-plansCardImage-9"
          height="500"
          src="PersonalPlan.png"
        />
        <div
          class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            class="MuiCardContent-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <p
                  class="MuiTypography-root makeStyles-plansCardHeader-10 MuiTypography-body1 MuiTypography-alignCenter"
                >
                  Personal Plan
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <p
                  class="MuiTypography-root makeStyles-plansCardText-11 MuiTypography-body1 MuiTypography-alignCenter"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root makeStyles-checkIconPersonal-12 MuiSvgIcon-fontSizeInherit"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                    />
                  </svg>
                  Pay what you like
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
                style="margin-bottom: 20px;"
              >
                <p
                  class="MuiTypography-root makeStyles-plansCardText-11 MuiTypography-body1 MuiTypography-alignCenter"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root makeStyles-checkIconPersonal-12 MuiSvgIcon-fontSizeInherit"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                    />
                  </svg>
                  Free for individuals
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <p
                  class="MuiTypography-root makeStyles-plansCardText-11 MuiTypography-body1 MuiTypography-alignCenter"
                >
                  Full access
                   
                  <span
                    class="makeStyles-plansCardText-11"
                    style="opacity: 0.8; border-radius: 5px; border: 1px solid #157ffb; background-color: rgb(21, 127, 251); color: white;"
                  >
                    for free
                  </span>
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <p
                  class="MuiTypography-root makeStyles-plansCardText-11 MuiTypography-body1 MuiTypography-alignCenter"
                  style="color: rgb(21, 127, 251);"
                >
                  <span
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-21 MuiRadio-root Component-root-19"
                  >
                    <span
                      class="MuiIconButton-label"
                    >
                      <input
                        class="PrivateSwitchBase-input-24"
                        type="radio"
                        value="personal"
                      />
                      <div
                        class="PrivateRadioButtonIcon-root-25"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                          />
                        </svg>
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root PrivateRadioButtonIcon-layer-26"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                          />
                        </svg>
                      </div>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </span>
                  This works for me
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
                >
                  <button
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-14 MuiLink-button MuiTypography-colorPrimary"
                    href="https://"
                  >
                    I would like to read more
                  </button>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root makeStyles-gridContent-18 MuiGrid-item MuiGrid-grid-xs-7 MuiGrid-grid-md-3"
      >
        <img
          alt="Illustration.png"
          class="makeStyles-plansCardImage-9"
          height="500"
          src="Illustration.png"
        />
        <div
          class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            class="MuiCardContent-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <p
                  class="MuiTypography-root makeStyles-plansCardHeader-10 MuiTypography-body1 MuiTypography-alignCenter"
                >
                  Business Plan
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <p
                  class="MuiTypography-root makeStyles-plansCardText-11 MuiTypography-body1 MuiTypography-alignCenter"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root makeStyles-checkIconBusiness-13 MuiSvgIcon-fontSizeInherit"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                    />
                  </svg>
                  License for Professionals
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
                style="margin-bottom: 20px;"
              >
                <p
                  class="MuiTypography-root makeStyles-plansCardText-11 MuiTypography-body1 MuiTypography-alignCenter"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root makeStyles-checkIconBusiness-13 MuiSvgIcon-fontSizeInherit"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                    />
                  </svg>
                  Â¢10 per
                   
                  <span
                    style="color: rgb(121, 30, 137);"
                  >
                    POP
                  </span>
                   
                  Prediction
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <p
                  class="MuiTypography-root makeStyles-plansCardText-11 MuiTypography-body1 MuiTypography-alignCenter"
                >
                  <span
                    class="makeStyles-plansCardText-11"
                    style="opacity: 0.8; border-radius: 5px; border: 1px solid #791e89; background-color: rgb(121, 30, 137); color: white;"
                  >
                    $10
                  </span>
                   
                  / month
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <p
                  class="MuiTypography-root makeStyles-plansCardText-11 MuiTypography-body1 MuiTypography-alignCenter"
                  style="color: rgb(121, 30, 137);"
                >
                  <span
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-21 MuiRadio-root Component-root-28"
                  >
                    <span
                      class="MuiIconButton-label"
                    >
                      <input
                        class="PrivateSwitchBase-input-24"
                        type="radio"
                        value="business"
                      />
                      <div
                        class="PrivateRadioButtonIcon-root-25"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                          />
                        </svg>
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root PrivateRadioButtonIcon-layer-26"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                          />
                        </svg>
                      </div>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </span>
                  This works for me
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
                >
                  <button
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-14 MuiLink-button MuiTypography-colorPrimary"
                    href="https://"
                  >
                    I would like to read more
                  </button>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
